Doc/library/sqlite3.rst

```diff
@@ -828,6 +828,12 @@ Exceptions
    disconnect occurs, the data source name is not found, a transaction could
    not be processed, etc.  It is a subclass of :exc:`DatabaseError`.
 
+.. exception:: NotSupportedError
+
+   Exception raised in case a method or database API was used which is not
+   supported by the database, e.g. requesting a .rollback() on a connection
```

Comment:
Perhaps we can replace "requesting a .rollback() on" with something like "calling the :meth:\`Connection.rollback\` method on" to make it sound more like stdlib documentation, not PEP 249?

Reply:
Fixed. Link works